docs/source/running-jobs/distributed-jobs.rst

```diff
@@ -185,6 +185,7 @@ To execute a distributed Ray program on many nodes, you can download the `traini
       else
         sleep 5
         ps aux | grep ray | grep 6379 &> /dev/null || ray start --address $HEAD_IP:6379 --disable-usage-stats
+        sleep 5
```

Comment:
Can we add a comment here? I think we accidently removed this sleep after the ray start, since it seems useless at a glance.